docs/modules/callbacks.rst

```diff
@@ -0,0 +1,26 @@
+Callbacks
+=========
+
+.. note::
+   `Conceptual Guide <https://docs.langchain.com/docs/components/callbacks>`_
```

Comment:
<https://docs.langchain.com/docs/components/callbacks>
Can you make it a relative link? Thanks

Reply:
Hi @leo-gan, as of the comment above, it doesn't work with a relative link since it's automatically generated on docs build-time, so there's nowhere to point to AFAIK, check other similar files e.g. https://github.com/hwchase17/langchain/blob/master/docs/modules/agents.rst

Reply:
But I indeed changed the rest of the URLs to be relative paths to the actual files instead, as you suggested in your comments above 🤗

Reply:
Yeah. It points to a different site. My bad :)

Reply:
LGTM